business/istio_config.go

```diff
@@ -471,5 +473,5 @@ func getUpdateDeletePermissions(k8s kubernetes.IstioClientInterface, namespace,
 			log.Errorf("Error getting permissions [namespace: %s, api: %s, resourceType: %s]: %v", namespace, api, resourceType, permErr)
 		}
 	}
-	return canUpdate || canPatch, canDelete
+	return canCreate || canUpdate || canPatch, canDelete
```

Comment:
it seems weird to "merge" create permission with update/patch boolean.
How is it used on the frontend? a "create" button is displayed only if we have "create" permission, or is it something different?

Reply:
Create/Update are being using in similar scenarios, so it makes sense merge them at the moment. IMO

Reply:
ok, but then maybe it should be ANDed rather than ORed, because if you have update rights but not create rights, you should not be able to reach the create screen.